diagnostics/ENSO_MSE/MSE/xx

```diff
@@ -0,0 +1 @@
+
```

Comment:
What does this file do?  Can it be deleted?

Reply:
it is deleted now